flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/sink/KafkaRecordSerializationSchema.java

```diff
@@ -85,4 +86,28 @@ default void open(
          */
         int[] getPartitionsForTopic(String topic);
     }
+
+    /**
+     * Creates a default schema builder to provide common building blocks i.e. key serialization,
+     * value serialization, partitioning; based on a static target topic.
+     *
+     * @param topic target topic for all records
+     * @param <T> type of incoming elements
+     * @return {@link KafkaRecordSerializationSchemaBuilder}
+     */
+    static <T> KafkaRecordSerializationSchemaBuilder<T> builder(String topic) {
+        return new KafkaRecordSerializationSchemaBuilder<T>(topic);
```

Comment:
Translate to constant `Function`? Then `KafkaRecordSerializationSchemaBuilder` just use one ctor and one invariant.